cometd-javascript/cometd-javascript-common/src/main/webapp/js/cometd/index.js

```diff
@@ -0,0 +1,30 @@
+import { CometD } from "./CometD.js";
+import { Transport } from "./Transport.js";
+import { RequestTransport } from "./RequestTransport.js";
+import { LongPollingTransport } from "./LongPollingTransport.js";
+import { CallbackPollingTransport } from "./CallbackPollingTransport.js";
+import { WebSocketTransport } from "./WebSocketTransport.js";
+import { Utils } from "./Utils.js";
+import { Z85 } from "./Z85.js";
+
+import { AckExtension } from "./AckExtension.js";
+import { BinaryExtension } from "./BinaryExtension.js";
+import { ReloadExtension } from "./ReloadExtension.js";
+import { TimeStampExtension } from "./TimeStampExtension.js";
+import { TimeSyncExtension } from "./TimeSyncExtension.js";
+
+export {
+  CometD,
+  Transport,
+  RequestTransport,
+  LongPollingTransport,
+  CallbackPollingTransport,
+  WebSocketTransport,
+  Utils,
+  Z85,
+  AckExtension,
+  BinaryExtension,
+  ReloadExtension,
+  TimeStampExtension,
+  TimeSyncExtension,
+};
```

Comment:
We also have this syntax 👇
```suggestion
export { CometD } from "./CometD.js";
export { Transport } from "./Transport.js";
export { RequestTransport } from "./RequestTransport.js";
export { LongPollingTransport } from "./LongPollingTransport.js";
export { CallbackPollingTransport } from "./CallbackPollingTransport.js";
export { WebSocketTransport } from "./WebSocketTransport.js";
export { Utils } from "./Utils.js";
export { Z85 } from "./Z85.js";

export { AckExtension } from "./AckExtension.js";
export { BinaryExtension } from "./BinaryExtension.js";
export { ReloadExtension } from "./ReloadExtension.js";
export { TimeStampExtension } from "./TimeStampExtension.js";
export { TimeSyncExtension } from "./TimeSyncExtension.js";
```